Misc/NEWS.d/next/Library/2020-01-08-23-25-27.bpo-39242.bnL65N.rst

```diff
@@ -0,0 +1,2 @@
+Updated the Gmane domain which is used for the :class:`~nntplib.NNTP`'s example
+news reader server
```

Comment:
Maybe mention the old and new domain. Please also add a final dot to the sentence. You may also mention that nntplib tests are also updated.